src/js/components/SelectMultiple/index.d.ts

```diff
@@ -8,6 +8,22 @@ export interface SelectMultipleProps extends BasicSelectProps {
   value?: (string | number | object)[];
   showSelectedInline?: boolean;
   sortSelectedOnClose?: boolean;
+  messages?: {
```

Comment:
Changes are also needed for `src/js/components/Grommet/index.d.ts` and `src/js/components/Grommet/propTypes.js`.

At some point maybe the message ts and proptypes for a component should be defined in one spot but not necessary for this PR.

Reply:
was added